### PR TITLE
XWIKI-11127 : Improve the "Last Author" field to display the author's avatar on AWM livetables

### PR DESCRIPTION
**Pull request by siddhpant**
Initial commit to check how it goes. 

Please review. If it's ok, then merge.

https://jira.xwiki.org/browse/XWIKI-11127

### CODE REVIEW

File: xwiki-platform-core/xwiki-platform-appwithinminutes/xwiki-platform-appwithinminutes-ui/src/main/resources/AppWithinMinutes/LiveTableGenerator.xml [hunk 1]
```diff
@@ -128,6 +128,10 @@ ${escapetool.h}set (${escapetool.d}columnsProperties = {
 #set($userPicker = true)
 ${escapetool.h}set (${escapetool.d}discard = ${escapetool.d}xwiki.ssfx.use('uicomponents/widgets/userpicker/userPicker.css'))
 #end
+#if($column == 'doc.author')
```
Comment (tmortagne) [inline xwiki-platform-core/xwiki-platform-appwithinminutes/xwiki-platform-appwithinminutes-ui/src/main/resources/AppWithinMinutes/LiveTableGenerator.xml]: It does not make much sense to have two different #if with the exact same content. You should merge them.

Comment (siddhpant) [reply xwiki-platform-core/xwiki-platform-appwithinminutes/xwiki-platform-appwithinminutes-ui/src/main/resources/AppWithinMinutes/LiveTableGenerator.xml]: Ikr I already committed it.

**Comment by tmortagne** [comment]
I doubt this is enough to get an avatar for the last author. Have you tested this ?

**Comment by mflorea** [comment]
It's not enough. The live table results (JSON) that feed the live table need to be modified also.

**Comment by siddhpant** [comment]
Or that livetable.js file in the livetable UI folder?

**Comment by siddhpant** [comment]
I meant this one:

https://github.com/xwiki/xwiki-platform/blob/ed89b3c1a8559852846c0e41d312d4204b458620/xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/table/livetable.js

**Comment by mflorea** [comment]
@siddhpant sorry for the late reply. Yes, I was referring to the ``XWiki.LiveTableResultsMacros`` page. Look for the ``gridresult_buildRowJSON`` macro which sets the document author on the response. There's a macro in ``macros.vm`` that you can use to display the user with his avatar.

**Pull request closed without merging**